warehouse/query-core/src/test/resources/input/generic-cities.csv

```diff
@@ -13,4 +13,5 @@
 20151010,ldn-usa-mi-10,London,Michigan,United States,North America,USA,NA,140
 20151111,par-ita-11,Paris,Lazio,Italy,Europe,ITA,Euro,30
 20151111,rom-bel-11,Rome,Hainaut,Belgium,Europe,BEL,Euro,50
-20151111,ldn-fra-lle-11,London,lle-de-France,France,Europe,FRA,Euro,20
\ No newline at end of file
+20151111,ldn-fra-lle-11,London,lle-de-France,France,Europe,FRA,Euro,20
+20151111,edge-case-id-1,\\Edge-City,Edgeville,Edgeland,Edgeope,EDG,NA,220
```

Comment:
lets add a single backslash and perhaps a triple backslash case and verify those

Reply:
I added the two cases you requested.  They work out fine, however I think this needs to bake a little more for other reasons...

Reply:
Added even more edge cases.  Check the latest commit.